po/af.po

```diff
@@ -32,21 +31,15 @@ msgstr "Programbestuurder vir GNOME"
 msgid ""
 "Software allows you to find and install new applications and system "
 "extensions and remove existing installed applications."
-msgstr ""
-"Sagteware laat mens nuwe programme en uitbreidings vind en installeer en kan "
-"ook reeds geïnstalleerde programme verwyder."
+msgstr "Sagteware laat mens nuwe programme en uitbreidings vind en installeer en kan ook reeds geïnstalleerde programme verwyder."
```

Comment:
Many of the changes in “Sync with transifex” appear to just be reformatting the line breaks. Perhaps this commit would also benefit from a `ninja gnome-software-update-po` command to normallise things again, unless that’s going to undo the transifex work?

Reply:
We could do. But – every time we run `tx pull -f` we'll get back Transifex's (lack of) line breaks. At some point in the past this was run automatically by Jenkins. https://ci.endlessm-sf.com/job/transifex_pull/ / https://ci.endlessm-sf.com/job/transifex_update/ – neither of these is run automatically at the moment but it would nice for the automation to return. Personally I think we should treat "what Transifex gives us" as the canonical form for our downstream translations.

It's really annoying that all these tools use different layouts for the same information. The best approach I have found for dealing with this is:

```
$ cat .gitattributes 
*.po	diff=podiff
*.pot	diff=podiff
$ git config diff.podiff.command
podiff -D -u
```

There are many things called `podiff` but the one I've used is http://man.gnu.org.ua/manpage/?1+podiff .